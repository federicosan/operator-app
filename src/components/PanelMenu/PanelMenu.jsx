import React, { Component } from 'react';
import { connect } from 'react-redux';

// DEPRECATED: Composition over Inheritence
export class PanelMenuComponent extends Component {
  constructor (props) {
    super(props);

    this.state = {};
  }

  render () {
    return (
      <div className="row">
        <ul className="menu" />
      </div>
    );
  }
}

const mapStateToProps = state => ({

});

const mapDispatchToProps = dispatch => ({

});

const PanelMenu = connect(
  mapStateToProps,
  mapDispatchToProps,
)(PanelMenuComponent);

export default PanelMenu;
