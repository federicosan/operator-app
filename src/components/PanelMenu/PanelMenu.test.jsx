import React from 'react';

import { shallow } from 'enzyme';

import PanelMenu from './PanelMenu.jsx';

const store = {
  subscribe: jest.fn(),
  dispatch: jest.fn(),
  getState: jest.fn(() => ({ })),
};

describe('PanelMenu', () => {
  it('matches snapshot', () => {
    const component = shallow(<PanelMenu store={store} />);

    expect(component).toMatchSnapshot();
  });
});
