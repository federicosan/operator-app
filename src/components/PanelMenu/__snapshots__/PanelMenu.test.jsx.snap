exports[`PanelMenu matches snapshot 1`] = `
<PanelMenuComponent
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "subscribe": [Function],
    }
  } />
`;
